com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricClouds.cs

```diff
@@ -347,15 +347,15 @@ public sealed class CloudMapResolutionParameter : VolumeParameter<CloudMapResolu
         /// <summary>
         /// Sets the global wind speed in kilometers per hour.
         /// </summary>
-        [Tooltip("Sets the global wind speed in kilometers per hour.")]
-        public MinFloatParameter globalWindSpeed = new MinFloatParameter(100.0f, 0.0f);
+        [Tooltip("Sets the global wind speed in kilometers per hour.\nThis value can be relative to the Global Wind Speed defined in the Visual Environment.")]
+        public WindSpeedParameter globalWindSpeed = new WindSpeedParameter();
```

Comment:
relativeWindSpeed with previously serialized?

Reply:
to simplify things i marked the old one as obsolete. It may break some code but i don't think a lot of people are using it in a script, do you think it's an issue ?